### PR TITLE
feat(stake-eth): add staking dashboard layout

### PR DESCRIPTION
**Pull request by StanCheb**
## Description
The staking dashboard layout is added to the ETH staking tab. 

NB! These are layouts only with some demo/test hardcoded values.

## Related Issue

Resolve 
https://github.com/trezor/trezor-suite/issues/9277
https://github.com/trezor/trezor-suite/issues/9274
https://github.com/trezor/trezor-suite/issues/9272
https://github.com/trezor/trezor-suite/issues/9268

## Screenshots:
Знімок екрана 2023-12-20 о 15 21 37

### CODE REVIEW

File: packages/components/src/components/Image/images.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const SVG_IMAGES = {
     PLAY_STORE_TITLE: 'play-store-title.svg',
     STROKE_BORDER: 'stroke-border.svg',
     TREZOR_SAFE_PROMO_UNDERLINE: 'trezor-safe-promo-underline.svg',
+    SPINNER_ORANGE: 'spinner-orange.svg',
```
Comment (tomasklim) [inline packages/components/src/components/Image/images.ts]: nit: move under `SPINNER`

File: packages/components/src/components/assets/Icon/icons.ts [hunk 2]
```diff
@@ -131,4 +131,7 @@ export const ICONS = {
     ARROW_BEND_DOUBLE_UP_LEFT: require('../../../images/icons/arrow_bend_double_up_left.svg'),
     HAND_COINS: require('../../../images/icons/hand_coins.svg'),
     LOCK_SIMPLE: require('../../../images/icons/lock_simple.svg'),
+    PERCENT: require('../../../images/icons/percent.svg'),
+    CALENDAR_THIN: require('../../../images/icons/calendar_thin.svg'),
+    PLUS_CIRCLE: require('../../../images/icons/plus_circle.svg'),
```
Comment (tomasklim) [inline packages/components/src/components/assets/Icon/icons.ts]: nit: Move under CIRCLE

I know it is not alphabetically sorted properly, but at least little bit it is, please try to add it to proper locations 🙏

File: packages/suite/src/components/suite/StakeAmountWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+const Container = styled.div`
+    position: relative;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    margin: -2px -12px;
+    padding: 2px 12px;
+    border-radius: 6px;
+    transition: background 0.1s ease-in;
+    cursor: pointer;
```
Comment (tomasklim) [inline packages/suite/src/components/suite/StakeAmountWrapper.tsx:17]: nit: I would take user to staking tab on click

File: packages/suite/src/support/messages.ts [hunk 4]
```diff
@@ -8503,4 +8503,53 @@ export default defineMessages({
...
+        defaultMessage: 'Stake more',
+    },
+    TR_STAKE_UNSTAKE_TO_CLAIM: {
+        id: 'TR_STAKE_UNSTAKE_TO_CLAIM',
+        defaultMessage: 'Unstake to claim',
+    },
+    TR_STAKE_ETH_REWARDS_EARN_APY: {
+        id: 'TR_STAKE_ETH_REWARDS_EARN_APY',
+        defaultMessage:
+            'Your ETH rewards also earn the APY rate. Keep your funds staked or add more to increase your rewards.',
```
Comment (tomasklim) [inline packages/suite/src/support/messages.ts]:
```suggestion
            'Your {symbol} rewards also earn the APY rate. Keep your funds staked or add more to increase your rewards.',
```

To be correct in testnets

File: packages/suite/src/support/messages.ts [hunk 5]
```diff
@@ -8503,4 +8503,53 @@ export default defineMessages({
...
+    },
+    TR_STAKE_REWARDS: {
+        id: 'TR_STAKE_REWARDS',
+        defaultMessage: 'Rewards',
+    },
+    TR_STAKE_ETH_AT_THE_DOOR: {
+        id: 'TR_STAKE_ETH_AT_THE_DOOR',
+        defaultMessage: 'Your ETH is at the door, waiting to join the staking pool.',
```
Comment (tomasklim) [inline packages/suite/src/support/messages.ts]:
```suggestion
        defaultMessage: 'Your {symbol} is at the door, waiting to join the staking pool.',
```

**Review by tomasklim — APPROVED** [review]
Great work! just nits feel free to fix them and merge it

File: packages/suite/src/support/messages.ts (hunk 5 above)
Comment (StanCheb) [reply packages/suite/src/support/messages.ts]: https://github.com/everstake/trezor-suite/pull/5/commits/25028a4faf3c04394eda3e5a7fee23435adbf3ef

File: packages/suite/src/support/messages.ts (hunk 4 above)
Comment (StanCheb) [reply packages/suite/src/support/messages.ts]: https://github.com/everstake/trezor-suite/pull/5/commits/25028a4faf3c04394eda3e5a7fee23435adbf3ef

File: packages/suite/src/components/suite/StakeAmountWrapper.tsx (hunk 3 above)
Comment (StanCheb) [reply packages/suite/src/components/suite/StakeAmountWrapper.tsx:17]: https://github.com/everstake/trezor-suite/pull/5/commits/25028a4faf3c04394eda3e5a7fee23435adbf3ef

File: packages/components/src/components/assets/Icon/icons.ts (hunk 2 above)
Comment (StanCheb) [reply packages/components/src/components/assets/Icon/icons.ts]: Got it. Moved everything that was added before in proper places.

https://github.com/everstake/trezor-suite/pull/5/commits/25028a4faf3c04394eda3e5a7fee23435adbf3ef

File: packages/components/src/components/Image/images.ts (hunk 1 above)
Comment (StanCheb) [reply packages/components/src/components/Image/images.ts]: https://github.com/everstake/trezor-suite/pull/5/commits/25028a4faf3c04394eda3e5a7fee23435adbf3ef

**Pull request merged**